sites/shared/components/account/sets.mjs

```diff
@@ -110,495 +109,486 @@ export const NewSet = ({
-        <button
-          className="btn btn-secondary btn-outline flex flex-row gap-4 rounded-r-none"
-          onClick={() =>
-            setModal(
-              <ModalDesignPicker
-                designs={Object.keys(designMeasurements)}
-                setModal={setModal}
-                setter={setFilter}
-              />
-            )
-          }
-        >
-          <FilterIcon />
-          {filter ? t(`designs:${filter}.t`) : t(`designs:allDesigns`)}
-        </button>
-        <button
-          className="btn btn-secondary btn-outline rounded-l-none border-l-0"
-          onClick={() => setFilter(false)}
-        >
-          <ClearIcon />
-        </button>
-      </div>
-      {filterMeasurements().map((m) => (
-        <MeasieRow key={m} m={m} {...props} {...saveProps} />
-      ))}
+      <button
+        onClick={save}
+        className="btn btn-primary btn-lg flex flex-row items-center gap-4 mx-auto mt-8"
+      >
+        <UploadIcon />
+        {t('saveThing', { thing: t('account:set') })}
+      </button>
     </div>
   )
 }
 
-const MeasurementsSet = ({ mset, t, account, backend, refresh }) => {
-  // Context
-  const { startLoading, stopLoading } = useContext(LoadingContext)
-  const { setModal } = useContext(ModalContext)
-
-  // Hooks
-  const toast = useToast()
-
-  const remove = async () => {
-    startLoading()
-    const result = await backend.removeSet(mset.id)
-    if (result) toast.success(t('gone'))
-    else toast.for.backendError()
-    // This just forces a refresh of the list from the server
-    // We obviously did not add a key here, but rather removed one
-    refresh()
-    stopLoading()
-  }
-
-  const removeModal = () => {
-    setModal(
-      <ModalWrapper slideFrom="top">
-        <h2>{t('areYouCertain')}</h2>
-        <p>{t('deleteSetWarning')}</p>
-        <p className="flex flex-row gap-4 items-center justify-center">
-          <button className="btn btn-neutral btn-outline px-8">{t('cancel')}</button>
-          <button className="btn btn-error px-8" onClick={remove}>
-            {t('delete')}
-          </button>
-        </p>
-      </ModalWrapper>
-    )
-  }
-
-  return (
-    <Collapse
-      primary
-      top
-      bottom
-      title={
-        <>
-          <img src={mset.img} className="w-10 mask mask-squircle bg-neutral aspect-square" />
-          <span>{mset.name}</span>
-        </>
-      }
-      openTitle={mset.name}
-      buttons={[
-        <button
-          key="rm"
-          className="btn btn-error hover:text-error-content border-0"
-          onClick={account.control > 4 ? remove : removeModal}
-        >
-          <TrashIcon key="button2" />
-        </button>,
-      ]}
-    >
-      <EditMeasurementsSet
-        {...{ t, mset, account, backend, toast, refresh, setModal, startLoading, stopLoading }}
-      />
-    </Collapse>
-  )
-}
-
 // Component for the account/sets page
 export const Sets = ({ title = true }) => {
```

Comment:
if these components don't rely on one another, I think it'd make sense for them to be in separate files because this file is overwhelmingly large

Reply:
no side-quests